### PR TITLE
Bugfix/startup order

### PR DESCRIPTION
**Pull request by AdozenBABYdogs**
**When merged this pull request will:**
- This merge prevents the starter from activating if the PCLs are not in the "OFF" position

### CODE REVIEW

**Review by BroBeansCPG — APPROVED** [review]
lgtm

**Comment by BroBeansCPG** [comment]
fixes #416

File: addons/uh60_sfmplus/functions/fn_standbyInst.sqf [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+params ["_heli", "_deltaTime"];
+
+private _ACBusState          = _vehicle getVariable "vtx_uh60_acft_ACBusState";
+private _stbyInstBatt        = _vehicle getVariable "vtx_uh60_acft_stbyInstBatt"; //1800 seconds
+private _stbyInstSwitchState = _vehicle getVariable "vtx_uh60_acft_stbyInstSwitchState";
+
+if (_ACBusState != "ON" && _stbyInstSwitchState == "ON") then {
+	if (_stbyInstBatt >= 0) then {
+		_stbyInstBatt = _stbyInstBatt - _deltaTime;
+		_vehicle setVariable ["vtx_uh60_acft_stbyInstBatt", _stbyInstBatt];
```
Comment (YannikH) [inline addons/uh60_sfmplus/functions/fn_standbyInst.sqf]: this is currently local to your machine, this should be shared over the network

File: addons/uh60_sfmplus/functions/fn_standbyInst.sqf [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	};
+};
+if (_ACBusState == "ON") then {
+	if (_stbyInstBatt < 1800) then {
+		_stbyInstBatt = _stbyInstBatt + _deltaTime;
+		_vehicle setVariable ["vtx_uh60_acft_stbyInstBatt", _stbyInstBatt];
```
Comment (YannikH) [inline addons/uh60_sfmplus/functions/fn_standbyInst.sqf]: this is currently local to your machine, this should be shared over the network

**Review by YannikH — CHANGES_REQUESTED** [review]

**Review by YannikH — APPROVED** [review]
LGTM

**Pull request merged**